rclcpp/include/rclcpp/exceptions/exceptions.hpp

```diff
@@ -206,6 +206,14 @@ class UnknownROSArgsError : public std::runtime_error
   const std::vector<std::string> unknown_ros_args;
 };
 
+/// Thrown when an unknown type is passed
+class UnkownTypeError : public std::runtime_error
```

Comment:
You mistyped `UnknownTypeError` above :grinning:

Reply:
```suggestion
class UnknownTypeError: public std::runtime_error
```